src/App.tsx

```diff
@@ -15,71 +16,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam is selected',
+  };
```

Comment:
`selectedGood: Jam is selected` looks really wrong. From the the name of the variable I expect that it's gonna be just `Jam`, but not the whole string to render. Let's keep only the good here